source/schemas/types.txt

```diff
@@ -509,3 +509,124 @@ with the ``title`` field set to the embedded object's type name:
             }
       }
    }
+
+.. _geospatial:
+
+Geospatial Data
+---------------
+
+Geospatial data describes points and other data on the earth's surface. The 
+Realm SDKs support the creation of numerous types of geospatial data. However, 
+only persisted GeoJSON Point data can be queried. 
+
+GeoJSON Point
+~~~~~~~~~~~~~
+
+A GeoJSON Point (GeoPoint) is a single location on the Earth's surface. The GeoPoint 
+schema must contain the following fields:
+
+- ``type`` (string), which is always set to "Point"
```

Comment:
[nit] I'm predicting the most common thing people will miss when they try to use GeoPoints is that "type" is a required field and must be marked as such in the JSON schema in order for the property to be picked up as a GeoPoint. Could we make that explicit here?

Reply:
@jonatlee to be clear, even if the schema doesn't require the coordinates field, it is also required in order to a) be compliant with the geopoint spec, and b) to do geowithin querying. Yes?

Reply:
yep that is correct. The schema can't have "coordinates" as a required field specifically because marking an array property as required in JSON schema will lead to the schema becoming incompatible with sync

Reply:
that being said^, sorry perhaps we should make it clear that it should only be the "type" field marked as required and "coordinates" not as a required in the JSON schema to prevent customers from accidentally creating sync incompatible schemas. it is true that both must be present however for those documents to actually pass validation though. 

as a side note, i was actually pushing for making "coordinates" a required field in the JSON schema and making required array fields sync compatible, but that was rejected 😅. i think the main reason is that it isn't immediately clear what "required" means for an array (ie, array length > 0, array is non-nil, etc)